packages/components/Menu/src/__tests__/__snapshots__/Menu.test.tsx.snap

```diff
@@ -194,6 +194,7 @@ Array [
               "maxWidth": 300,
               "minHeight": 32,
               "minWidth": 160,
+              "padding": 6,
```

Comment:
Hmm any ideas why this got added?

Reply:
This test is running on Android where sNudge was undefined before, so it makes sense that it's showing up now